js/modules/k6/ws/ws_test.go

```diff
@@ -732,9 +732,6 @@ func TestErrors(t *testing.T) {
 		if (!closed) {
 			throw new Error ("conn no closed");
 		}
-		if (!onerror) {
-			throw new Error ("error event has not triggered");
-		}
```

Comment:
This following test doesn't seem to do what the name says. I don't know how we could refactor it to simulate this case. All the cases I can think are not directly generated from the `socket.close` method. If we don't have better suggestions then I think we should drop it.

https://github.com/grafana/k6/blob/45dde486be59f751a5888f96cc5707ab153ec7cc/js/modules/k6/ws/ws_test.go#L694-L703